25-inf-model-mlr.Rmd

```diff
@@ -275,7 +275,7 @@ terms_chp_25 <- c(terms_chp_25, "multicollinearity")
 ```
 
 Although diving into the details are beyond the scope of this text, we will provide one more reflection about multicollinearity.
-If the predictor variables have some degree of correlation, it can be quite difficult to interpret the value of the coefficient or its significance.
+If the predictor variables have some degree of correlation, it can be quite difficult to interpret the value of the coefficient or its impact.
```

Comment:
```suggestion
If the predictor variables have some degree of correlation, it can be quite difficult to interpret the value of the coefficient or evaluate whether the variable is a statistically significant predictor of the outcome.
```